docs/docs/contributions/releases/release-process.mdx

```diff
@@ -158,6 +158,26 @@ Once the workflow finishes, look through any failures and determine if there's a
 
 Alternatively, after starting the workflow, post the link to the in-progress run in `#development` in Slack, so that someone can come back to it when it does finish.
 
+## Step 5: Publish a schema in JSON Schema Store
+
+Pants configuration file schema is published at https://www.schemastore.org/. For every release, a new schema needs to generated
+and uploaded by submitting a PR against https://github.com/SchemaStore/schemastore.
+
+To produce `pantsbuild-<version>.json` schema file, run:
+
+```bash
+pants help-all > all-help.json
+pants run build-support/bin/generate_json_schema.py -- --all-help-file=all-help.json
+```
+
+It may be helpful to compare the last schema file with the newly produced to make sure there are no discrepancies
```

Comment:
Where do we get the last schema file for this comparison?

Reply:
Updated the docs to explain!